<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatClient</class>
 <widget class="QWidget" name="ChatClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="logGroupBox">
      <property name="title">
       <string>Log In</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="logInUpSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>50</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="label_pic">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="logformLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="ipAddressLabel">
           <property name="text">
            <string>IP Address</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="ipAddressLineEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="portNumLabel">
           <property name="text">
            <string>Port No</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="portNumLineEdit"/>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="nameLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="logPushButtonLayout">
         <item>
          <widget class="QPushButton" name="logInPushButton">
           <property name="text">
            <string>Log In</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="logOutPushButton">
           <property name="text">
            <string>Log Out</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="logInUnderSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>115</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="chattingGroupBox">
      <property name="title">
       <string>Chatting</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="chatInPushButton">
         <property name="text">
          <string>Chat In</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="messageTextEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="messageLineEdit"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="chattingPushButtonLayour">
         <item>
          <widget class="QPushButton" name="fileTransferPushButton">
           <property name="text">
            <string>File Transfer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sendPushButton">
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
